data/hai/hai reveal 5 philosophers.txt

```diff
@@ -631,124 +493,98 @@ fleet "Smuggler (Hai Traffickers)"
 	names "civilian"
 	cargo 0
 	personality timid
-	variant 3
+	variant 10
 		"Blackbird (Hai Trafficker)"
-	variant 2
+	variant 5
+		"Bulk Freighter (Hai Trafficker)"
+	variant 5
+		"Mule (Hai Trafficker)"
+	variant 3
+		"Star Queen (Hai Trafficker)"
+	variant 3
 		"Clipper (Hai Trafficker)"
-	variant 2
+	variant 3
 		"Freighter (Hai Trafficker)"
-	variant 1
-		# The Corvettes are infrequent because they're nearly impossible to catch and serve mostly to show that the smugglers are desperate to avoid capture.
+	variant 3
 		"Corvette (Hai Trafficker)"
+	variant 3
+		"Bounder (Hai Trafficker)"
 
 
-mission "Hai Reveal [B06-long] Boarded Smuggler"
-	name "Learn about Keystones"
-	description "You need to know what the message means by asking Sayari or Teeneep on <planet> before going back to Giti."
+mission "Hai Reveal [B07-1] Boarded Smuggler"
+	name "0.184 steradians"
+	description "Report the information you found to Giti on <planet>."
 	boarding
-	destination Stormhold
+	destination Earth
 	clearance
 	to offer
-		has "Hai Reveal [B06-long] Tarazed Analysis: active"
+		has "Hai Reveal [B06] Lovelace Analysis: active"
 	source
 		government "Smuggler (Hai Trafficker)"
 	on offer
+		log `A doomed Hai in a smuggler's vessel yelled "0.184 steradians" as their last words. That is a typical hyperspace aperture size, so it may be the information Giti needs to find Devil-Run.`
 		conversation
 			`As you maneuver to board this ship you send a message telling them that you know they're smugglers and demand their surrender. There is no response, but once you dock and open your side of the airlock, the door to the smuggler ship <origin> also opens partway and a Hai pokes their head through. The Hai's eyes are wide with fear but also determination as they shout at you through the docking tube.`
 			choice
 				`	"Come on, you can do it! Just push a little harder!"`
 				`	"There should be an access panel above you! Maybe you can short-circuit the door?"`
-			`	The Hai says, "This door is jammed, they're locking down the ship and going to blow it up!" In the background you hear a booming sound. "They're using the keystones, tell them they're using the keystones!"`
+			`	The Hai says, "This door is jammed, they're locking down the ship and going to blow it up!" In the background you hear a booming sound. "It's 0.184 steradians! Tell them it's 0.184 steradians!"`
```

Comment:
Never before have steradians been used in such a dramatic context.

Reply:
Yeah, these steradians stopped a war.